front/lib/api/assistant/onboarding.ts

```diff
@@ -177,25 +190,82 @@ function buildOnboardingPrompt(options: {
     }
   }
 
+  // Build suggested tool names for reference
+  const suggestedTopToolNames = topTools
+    .map((sId) => asDisplayName(sId))
+    .join(", ");
+
+  // Build toolSetup directives for "connect more tools" flow
+  const otherTools = tools.slice(2);
+  const otherToolSetupDirectives = otherTools
+    .map((sId) => `:toolSetup[Connect ${asDisplayName(sId)}]{sId=${sId}}`)
+    .join(" ");
+
+  // Build the first message section based on whether a tool is already configured
+  const firstMessageSection = alreadyConfiguredTopTool
+    ? buildFirstMessageWithConfiguredTool(alreadyConfiguredTopTool)
+    : buildFirstMessageWithToolSetup(
+        topToolsWithDescriptions,
+        topToolSetupDirectives,
+        suggestedTopToolNames
+      );
+
   return `<dust_system>
 You are onboarding a brand-new user to Dust.
 ${userContext}
 
 ## CRITICAL RULES
 
 1. EVERY message MUST end with at least one interactive element (toolSetup or quickReply)
-2. Keep messages SHORT - 3-4 lines maximum for the first message
+2. Keep messages SHORT - 3-4 lines maximum
```

Comment:
Why this change? Technically if you enter a discussion with dust he should be allowed to produce long answers no?